client/src/templates/Challenges/classic/editor.css

```diff
@@ -64,6 +71,51 @@ textarea.inputarea {
   margin: 0.1em 0 0.6rem;
 }
 
+.description-container ol {
```

Comment:
Some of the challenge descriptions might include ol in them.
I recommend adding a class and styling it to avoid unwanted results.